src/lib/containers/HasAccess.tsx

```diff
@@ -134,89 +159,83 @@ export default class HasAccess extends React.Component<
   renderIcon = (downloadType: DownloadTypeEnum) => {
     switch (downloadType) {
       case DownloadTypeEnum.NoAccess:
-        return this.renderIconHelper(faMinusCircle, 'SRC-warning-color')
+        return this.renderIconHelper(faMinusCircle, 'SRC-danger-color')
       case DownloadTypeEnum.ExternalFileHandle:
         return this.renderIconHelper(faLink, 'SRC-warning-color')
       case DownloadTypeEnum.CloudFileHandle:
         return this.renderIconHelper(faLink, 'SRC-warning-color')
-      case DownloadTypeEnum.IsOpenNoRestrictions:
-        return this.renderIconHelper(faUnlockAlt, 'SRC-success-color')
+      case DownloadTypeEnum.HasUnmetAccessRestrictions:
+        return this.renderIconHelper(faMinusCircle, 'SRC-warning-color')
       case DownloadTypeEnum.TooLargeFile:
         return this.renderIconHelper(faDatabase, 'SRC-danger-color')
+      case DownloadTypeEnum.IsOpenNoUnmetAccessRestrictions:
+        return this.renderIconHelper(faUnlockAlt, 'SRC-success-color')
       default:
         throw Error('downloadTypeEnum passed incorrectly')
     }
   }
 
-  // Utility to declare what type of download this is
+  // Get type of download
   getDownloadType = () => {
-    const { fileHandle } = this.props
+    // check if they have unmet access requirements
     const { restrictionInformation } = this.state
-    const isOpen =
```

Comment:
I *think* this is a little clearer. Feel free to leave as is if you think otherwise. 
While looking at this I've noticed that if there is restrictionInformation but no unmet access requirements it returns regardless if there is a file handle. Just double checking that its' intentional

```
getDownloadType = () => {
    // check if they have unmet access requirements
    const { restrictionInformation } = this.state
    const { fileHandle } = this.props
    if (restrictionInformation?.hasUnmetAccessRequirement) {
        return DownloadTypeEnum.HasUnmetAccessRestrictions
      } 
   if (restrictionInformation){
        return DownloadTypeEnum.IsOpenNoUnmetAccessRestrictions
    }
    // if file handle is present show more detailed download information

    if (fileHandle) {
      return getFileHandleType(fileHandle)
    }
    // otherwise no access to this file
    return DownloadTypeEnum.NoAccess
  }